src/Hive.IO/Hive.IO/Results/Results.cs

```diff
@@ -789,28 +792,28 @@ public static double GetTotalGainsOrLosses(Building.Building building, string lo
                 switch (loadType)
                 {
                     case "Qt_opaque_positive":
-                        result += (from x in zone.OpaqueTransmissionHeatLosses where x > 0.0 select x).Sum();
+                        result += (from x in zone.OpaqueTransmissionHeatLossesMonthly where x > 0.0 select x).Sum();
```

Comment:
might be simpler with other form of LINQ like

`zone.OpaqueTransmissionHeatLossesMonthly.Where(x => x > 0.0).Sum()`

Reply:
ok we can do! d0a6533